android/app/src/main/AndroidManifest.xml

```diff
@@ -36,6 +37,14 @@
                 <category android:name="android.intent.category.LAUNCHER" />
                 <action android:name="android.intent.action.DOWNLOAD_COMPLETE" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW"/>
+                <category android:name="android.intent.category.DEFAULT"/>
+                <category android:name="android.intent.category.BROWSABLE"/>
+                <data android:host="lndr.page.link"/>
```

Comment:
I can see that Google controls the domain `page.link`. Is registering an intent to handle links on a subdomain of that domain required, or can we register this intent for a domain we control (e.g: `lndr.blockmason.link` or `mobile.lndr.io`)? Maybe you can explain to me a little about how deep-linking works?